tests/vxlan-non-zero-reserved-fields/test.yaml

```diff
@@ -0,0 +1,17 @@
+args:
```

Comment:
Since this change is for master, for now, let's add a `min-version: 8.0` requirement here. (as seen in https://github.com/OISF/suricata-verify/blob/master/tests/datajson/datajson-01-ip/test.yaml#L2)

Reply:
OK.